learner.py

```diff
@@ -2,12 +2,13 @@
 # OBJECT = un insieme di dati e metodi
 # CLASS = istruzioni per OBJECT
 
-"""Creo una classe/funzione 'Learner' dove inserirò tutte le informazioni degli studenti di cui ho bisogno
-(nome, cognome, livello di preparazione, classe di preparazione)."""
-class Learner (object): #le classi si scrivono in cammel case->LearnerFederico
+class Learner (object): #le classi si scrivono in camel case->LearnerFederico
+    """Creo una classe/funzione 'Learner' dove inserirò tutte le informazioni degli studenti di cui ho bisogno
```

Comment:
Sì, anche perché così le docstring diventano parte della documentazione della funzione o della classe. Mentre solitamente i commenti vanno prima o in linea con quello che stai commentando